src/getCustomLibp2p.ts

```diff
@@ -80,7 +80,7 @@ export async function getCustomLibp2p ({ datastore }: HeliaGatewayLibp2pOptions)
     },
     transports: [
       circuitRelayTransport({
-        discoverRelays: USE_LIBP2P ? 1 : 0
+        discoverRelays: 1
```

Comment:
We only need to discover relays if we want to be dialable over WebRTC.  We don't need to do it if, for example, we just want to be able to dial other peers over WebRTC.

This can probably be removed or otherwise set to 0?